plugins/modules/postgresql_info.py

```diff
@@ -1007,8 +1007,8 @@ def get_db_info(self):
         # Following query returns:
         # Name, Owner, Encoding, Collate, Ctype, Access Priv, Size
         query = ("SELECT d.datname, "
-                 "pg_catalog.pg_get_userbyid(d.datdba) username, "
-                 "pg_catalog.pg_encoding_to_char(d.encoding) encoding, "
+                 "pg_catalog.pg_get_userbyid(d.datdba) AS username, "
+                 "pg_catalog.pg_encoding_to_char(d.encoding) AS encoding, "
                  "d.datcollate, "
                  "d.datctype, "
                  "pg_catalog.array_to_string(d.datacl, E'\n') aclstring, "
```

Comment:
```suggestion
                 "pg_catalog.array_to_string(d.datacl, E'\n') AS aclstring, "
```
should this be also `AS`?

Reply:
For reasons of consistency one could argue "_yes!_", but there are a lot more places, if that's the plan. Also in terms of capitalization (`AS` vs `as`).
But technically it shouldn't be required as `aclstring` is no keyword in PostgreSQL, that's why I addressed `encoding` only in the original commit of this PR because I wanted to keep the change minimal and easy to review.

Reply:
@cschug ah, great, sure, i see now. Thanks! Marking as resolved